traffic_ops/testing/api/v3/traffic_control_test.go

```diff
@@ -59,3 +59,11 @@ type TrafficControl struct {
 	Users                                             []tc.User                               `json:"users"`
 	InvalidationJobs                                  []tc.InvalidationJobInput               `json:"invalidationJobs"`
 }
+
+// NegativeFixtures - maps to the tc-negative-fixtures.json file
```

Comment:
GoDoc should be a complete sentence starting with the documented entity and ending with punctuation.

Reply:
Made this into a sentence in 5bd6414471.